app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -59,7 +59,7 @@ private void nextActivity() {
             LOGGER.info("Sending my profile with name: " + myProfile.getName() + ", public key: "
                     + myProfile.getPublicKey().toString());
             final Contact myProfileAsContact = new Contact(myProfile.getName(), new SimpleKey("simplekey", "73890ien"));
-            this.serviceLocator.getBluetoothHandler().send(myProfileAsContact);
+            this.serviceLocator.getBluetoothHandler().send(this.serviceLocator.getBluetoothHandler().objectToBytes(myProfileAsContact));
```

Comment:
> Yes, I can. NFC needs the message as bytes in order to send it. However, it cannot use the send() method of APairingHandler

I was not aware of that. So OK, the `objectToBytes()` methods must be exposed, but you can still have two `send()` methods (or even one, not sure if we really need to) where one of them accepts any object and uses the `objectToBytes()` methods to get the bytes of that object